### PR TITLE
Send 1% of traffic to gesis

### PR DESCRIPTION
**Pull request by betatim**
This will setup the binder federation to send 1% of traffic to the gesis instance.

### CODE REVIEW

**Comment by choldgraf** [comment]
:-D

**Comment by betatim** [comment]
Waiting to figure out why deploys to OVH aren't working right now. After that @bitnik and I will try and add Gesis. Let's see what happens.

File: mybinder/values.yaml [hunk 1]
```diff
@@ -425,5 +425,9 @@ federationRedirect:
       prime: true
     ovh:
       url: https://ovh.mybinder.org
-      weight: 15
+      weight: 14
       health: https://ovh.mybinder.org/health
+    gesis:
+      url: https://notebooks.gesis.org/binder/
```
Comment (betatim) [inline mybinder/values.yaml]:
```suggestion
      url: https://notebooks.gesis.org/binder
```
via @bitnik

**Comment by bitnik** [comment]
nice :)

https://notebooks.gesis.org/grafana/d/fZWsQmnmz/pod-activity?refresh=1m&panelId=3&orgId=1 -> here we can monitor pod activity

**Pull request merged**